ocaml/graph/go

```diff
@@ -1,13 +1,17 @@
 #!/bin/sh
 
+set -e
+
+GRAPH="../../graph.native -h $1 -u $2 -pw $3"
 DOTARGS="-Tpng -Ktwopi -Goverlap=false"
-GRAPH=./graph
 
 ${GRAPH} VM VBD VDI SR > vm-disks.dot
 ${GRAPH} VM VIF network > vm-network.dot
 ${GRAPH} host PIF network > host-network.dot
 
-for i in vm-disks.dot vm-network.dot host-network.dot
+for i in vm-disks vm-network host-network
 do
-  dot ${DOTARGS} $i > $i.png
+  dot ${DOTARGS} $i.dot > $i.png
 done
+
+set +e
```

Comment:
What is the significance of having this at the end of the file? (I'm not an expert at bash, I usually just put a `set -uex` to the beginning of the script).